### PR TITLE
[NO-TICKET] Update package script documentation in the README

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

Updated package script documentation in the README. Some entries were missing, and some were outdated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,13 +40,21 @@ These scripts can all be run from the root level of the repo:
...
-  - Builds the docs site statically
-
+  - Compiles everything and makes things release-ready
+  - Building is required to get TypeScript completion for the core package in child design system packages
+- `yarn build:{core,cmsgov,healthcare,medicare}`
+  - Builds a specific brand of the design system and its corresponding package
+- `yarn build:docs`
+  - Statically builds a copy of the doc site
+  - You can then serve it with `yarn serve:docs`
+- `yarn build:storybook`
```
Comment (zarahzachz) [inline README.md:50]: Is `yarn build:storybook:docs` still a thing? I see it in package.json, but not described here.

Comment (zarahzachz) [reply README.md:50]: As an aside, should `yarn build:storybook:docs` actually be `yarn build:storybook-docs` to match the testing command?

**Review by zarahzachz — APPROVED** [review]
One question, but looks good

File: README.md (hunk 1 above)
Comment (pwolfert) [reply README.md:50]: It's only used by other scripts to build storybook specifically for the docs. It isn't building "storybook docs" but rather building "storybook" for the "docs" package (Gatsby site). It's not something people should have to manually run, so I didn't want to explain it here.

**Pull request merged**